### PR TITLE
Add Ping() to v2 client.

### PR DESCRIPTION
**Pull request by PSUdaemon**
This was in the original client but did not make it to the v2 client. I think it's useful and have provided this patch.

Thanks.

### CODE REVIEW

File: client/v2/client.go [hunk 1]
```diff
@@ -119,6 +123,49 @@ func NewHTTPClient(conf HTTPConfig) (Client, error) {
 	}, nil
 }
 
+// Pings cluster
+func (c *client) Ping(timeout time.Duration) (time.Duration, string, error) {
```
Comment (e-dard) [inline client/v2/client.go]: `golint` won't like this comment. Can you put something more descriptive, beginning with `Ping...`. Possibly take your description on the interface and put it here, with a short description on the interface.

File: client/v2/client.go [hunk 2]
```diff
@@ -149,6 +196,11 @@ func NewUDPClient(conf UDPConfig) (Client, error) {
 	}, nil
 }
 
+// Pings cluster
+func (uc *udpclient) Ping(timeout time.Duration) (time.Duration, string, error) {
```
Comment (e-dard) [inline client/v2/client.go]: Comment needs to begin with `Ping...`

File: client/v2/client.go (hunk 1 above)
Comment (PSUdaemon) [inline client/v2/client.go]: I don't mind making this change and in fact I already have it in my tree, but I went to confirm that `golint` was happy and it seems it doesn't actually care about the plural form. Are there some options I need to use to get it to be strict about that?

    $ golint
    client.go:24:6: exported type HTTPConfig should have comment or be unexported
    client.go:50:6: exported type UDPConfig should have comment or be unexported
    client.go:60:6: exported type BatchPointsConfig should have comment or be unexported
    client.go:91:1: comment on exported function NewHTTPClient should be of the form "NewHTTPClient ..."
    client.go:321:6: exported type Point should have comment or be unexported
    client.go:364:1: comment on exported method Point.Tags should be of the form "Tags ..."

Let me know if I should push my updated comments for consistency. I can also fix the other comments in a separate PR.

Thanks.

Comment (e-dard) [inline client/v2/client.go]: I guess my comment was meant more as "put the guts of the description on the method doc, and the shorter description on the interface field" :smile: 

And yes, we always appreciate some `golint` tidying!

**Comment by e-dard** [comment]
LGTM :+1:

**Comment by jwilder** [comment]
@sparrc can you take a look?

**Comment by sparrc** [comment]
LGTM :+1:

**Comment by jwilder** [comment]
Thanks @PSUdaemon

**Comment by dneuman64** [comment]
:+1:

**Pull request merged**